tests/simpleharness/GetAllocatedBytesForCurrentThreadTest.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Microsoft.Xunit.Performance;
+using System;
+using System.Reflection;
+
+namespace simpleharness
+{
+    public class GetAllocatedBytesForCurrentThreadTest
+    {
+        [Benchmark]
+        public static void WithCollect()
+        {
+            var method = typeof(GC)
+                .GetTypeInfo()
```

Comment:
The `tests/standalonesample` is the example of a simpleharness. The latter is a project with direct project references to the API (a non-user scenario).
I will be happy to clean this up on subsequent PR =)